### PR TITLE
Update bug report template to include personas

### PR DESCRIPTION
**Pull request by mikang**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -34,5 +34,10 @@ If applicable, add screenshots to help explain your problem.
  - Browser: [e.g. stock browser, safari]
  - Version: [e.g. 22]
 
+**For whom is this feature?**
```
Comment (dmundra) [inline .github/ISSUE_TEMPLATE/bug_report.md:37]: I think this should be a bug question. E.g.

> While [activity] as [persona], I expected [result] but observed [result].

**Review by dmundra — CHANGES_REQUESTED** [review]
I think we need some small rephrasing of the text in the bug template.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (dmundra) [reply .github/ISSUE_TEMPLATE/bug_report.md:37]: And the bold section could say

> **For whom does this bug occur?**

Comment (mikang) [reply .github/ISSUE_TEMPLATE/bug_report.md:37]: Nice! I've made the edit and submitted another pull request. I'm going to close out this one, as I don't know how to edit the settings in a pull request.

Comment (dmundra) [reply .github/ISSUE_TEMPLATE/bug_report.md:37]: Ah, no worries. I think you can edit a pull request like before, which is in the main repo switch the branch to this pull request's branch and make the changes and submit it.

**Pull request closed without merging**